meteor/client/ui/RundownView/RundownTiming/PartCountdown.tsx

```diff
@@ -7,29 +7,29 @@ import { RundownUtils } from '../../../lib/rundown'
 interface IPartCountdownProps {
 	partId?: PartId
 	hideOnZero?: boolean
+	label?: ReactNode
 }
 
 /**
  * A presentational component that will render a countdown to a given Part
- * @class PartCountdown
+ * @function PartCountdown
  * @extends React.Component<WithTiming<IPartCountdownProps>>
  */
-export const PartCountdown = withTiming<IPartCountdownProps, {}>()(
-	class PartCountdown extends React.Component<WithTiming<IPartCountdownProps>> {
-		render() {
-			return (
-				<span>
-					{this.props.partId &&
-						this.props.timingDurations &&
-						this.props.timingDurations.partCountdown &&
-						this.props.timingDurations.partCountdown[unprotectString(this.props.partId)] !== undefined &&
-						(this.props.hideOnZero !== true ||
-							this.props.timingDurations.partCountdown[unprotectString(this.props.partId)] > 0) &&
-						RundownUtils.formatTimeToShortTime(
-							this.props.timingDurations.partCountdown[unprotectString(this.props.partId)]
-						)}
-				</span>
-			)
-		}
-	}
-)
+export const PartCountdown = withTiming<IPartCountdownProps, {}>()(function PartCountdown(
+	props: WithTiming<IPartCountdownProps>
+) {
+	if (!props.partId || !props.timingDurations?.partCountdown) return null
+
+	const shouldShow =
+		props.timingDurations.partCountdown[unprotectString(props.partId)] !== undefined &&
```

Comment:
Since `props.timingDurations.partCountdown[unprotectString(props.partId)]` is repeated three times it could be put in a const for better readability perhaps?

Reply:
Yeah, good point!